packages/easy-hydrator/src/ArrayToValueObjectHydrator.php

```diff
@@ -41,12 +38,10 @@ public function hydrateArray(array $data, string $class): object
         /** @var CacheItem $cacheItem */
         $cacheItem = $this->filesystemAdapter->getItem($arrayHash);
         if ($cacheItem->get() !== null) {
-            return $cacheItem->get();
+            // return $cacheItem->get();
```

Comment:
This needs revert or fix

Reply:
Good catch. I intend to implement null cache provider for tests in another pr. Forget this one 👍